docs/src/release-notes/v0_17.md

```diff
@@ -44,6 +44,14 @@ You can also look at the same information in a flamegraph by clicking the little
 
 Special thanks to [Valentin](https://github.com/vchuravy) and [Connor](https://github.com/connor4312) for their help with this feature!
 
+## Inline result styling improvements
+
+The stylings of inline result bubbles are greatly improved;
+the left-border color will make it easier to distinguish them from the actual code:
+
+- in dark theme ([Iceberg](https://marketplace.visualstudio.com/items?itemName=harg.iceberg)): ![inline bubbles dark](./0.17/inline_bubbles_dark.png)
+- in light theme ([Night Owl Light](https://marketplace.visualstudio.com/items?itemName=sdras.night-owl)): ![inline bubbles light](./0.17/inline_bubbles_light.png)
+
```

Comment:
If you think this section is verbose, I don't mind if you remove it

Reply:
I think the more we write the better :)